pytorch_lightning/metrics/__init__.py

```diff
@@ -56,4 +57,10 @@
     "SSIM"
 ]
 __sequence_metrics = ["BLEUScore"]
-__all__ = __regression_metrics + __classification_metrics + ["SklearnMetric"] + __sequence_metrics
+__selfsuper_metrics = ["EmbeddingSimilarity"]
+
+__all__ = __regression_metrics \
```

Comment:
one question here: do we really have to define them here or can't we just import them from the respective file's __all__? (Something like `from pytorch_lightning.metrics.regression import __all__ as __regression_metrics`).

Also don't we have the __all__ defined in all the metrics files? If so we could just do from `pytorch_lightning.metrics.classification import *`

Reply:
I have changed it to your suggestion, such that each file defines its own `__all__` which we then import in `__init__` and concatenate into one `__all__` for the hole package